pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -1050,30 +1050,20 @@ pub mod pallet {
 			Self::ensure_for_mint(player, mint_option)?;
 
 			let season_id = CurrentSeasonStatus::<T>::get().season_id;
-			let season = Seasons::<T>::get(season_id).ok_or(Error::<T>::UnknownSeason)?;
-			let mint_output =
-				mint_option.mint_version.with_minter(|minter: Box<dyn Minter<T>>| {
-					minter.mint_avatar_set(player, &season_id, &season, mint_option)
-				})?;
-
-			// Add generated avatars from minter to storage
-			let generated_avatar_ids = mint_output
-				.into_iter()
-				.map(|(minted_avatar_id, minted_avatar)| {
-					Self::try_add_avatar_to(player, minted_avatar_id, minted_avatar)?;
```

Comment:
@DidacSF Was there a reason we had to move database operations here?

Reply:
Hm, no particular reason I think